### PR TITLE
chore: Rename Hydroflow -> DFIR

### PR DESCRIPTION
**Pull request by rohitkulshreshtha**
Work In Progress:
- [x] hydroflow_macro
- [x] hydroflow_datalog_core
- [x] hydroflow_datalog
- [x] hydroflow_lang
- [x] hydroflow

### CODE REVIEW

**Comment by shadaj** [comment]
The diff seems to introduce a lot new lines without removing old
ones (same for the HF+ PR), are there leftover files somewhere?

**Comment by rohitkulshreshtha** [comment]
> The diff seems to introduce a lot new lines without removing old ones (same for the HF+ PR), are there leftover files somewhere?

Not sure. Files have been moved - how are those accounted in statistics?

**Comment by shadaj** [comment]
Moves shouldn't count. My hunch is there are new snapshot files but old ones haven't been cleaned up? Because snapshot file names include the crate name.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -86,8 +86,8 @@ jobs:
           --no-changelog-preview --allow-fully-generated-changelogs
           --bump ${{ inputs.bump }} --bump-dependencies auto
           ${{ inputs.execute && '--execute' || '--no-publish' }}
-          hydroflow hydroflow_lang hydroflow_macro hydroflow_plus
-          hydroflow_plus_std hydroflow_datalog hydroflow_datalog_core
+          hydroflow dfir_lang dfir_macro hydroflow_plus
+          hydroflow_plus_std dfir_datalog dfir_datalog_core
```
Comment (MingweiSamuel) [inline .github/workflows/release.yml:90]: Seems the hydroflow_plus ones here didn't get updated

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -10,11 +10,11 @@ The Hydroflow repo is set up as a monorepo and [Cargo workspace](https://doc.rus
 Relative to the repository root:
 
 * `hydroflow` is the main Hydroflow package, containing the Hydroflow runtime. It re-exports the
```
Comment (MingweiSamuel) [inline CONTRIBUTING.md:12]: `dfir_rs`

**Comment by MingweiSamuel** [comment]
Oops - this didn't go into main

**Comment by MingweiSamuel** [comment]
3291c07b37c9f9031837a2a32953e8f8854ec298

**Pull request merged**